src/hooks/http.test.js

```diff
@@ -188,7 +280,7 @@ describe('http hook', () => {
     const wrapper = Http.httpRequestWriteBeforeHookWrapper(requestData);
     wrapper([firstArg, secArg]);
 
-    expect(requestData).toEqual({ body: 'BODY' });
```

Comment:
Add some test with `truncated=true`

Reply:
there are tests for this elsewhere in reporter.test

 test('forgeAndScrubRequestBody truncated response', () => {